### PR TITLE
Minor grammar corrections in bundle generator command

### PR DESCRIPTION
**Pull request by darrylhein**

### CODE REVIEW

File: Command/GenerateBundleCommand.php [hunk 1]
```diff
@@ -150,7 +150,7 @@ protected function interact(InputInterface $input, OutputInterface $output)
                 'See http://symfony.com/doc/current/cookbook/bundles/best_practices.html#bundle-name for more',
                 'details on bundle naming conventions.',
                 '',
-                'Use <comment>/</comment> instead of <comment>\\ </comment> for the namespace delimiter to avoid any problem.',
+                'Use <comment>/</comment> instead of <comment>\\</comment> for the namespace delimiter to avoid any problems.',
```
Comment (HeahDude) [inline Command/GenerateBundleCommand.php]: the space after the backslash should be reverted and the one before `for` removed instead, or `<` will be escaped breaking the style.

File: Command/GenerateBundleCommand.php [hunk 2]
```diff
@@ -150,7 +150,7 @@ protected function interact(InputInterface $input, OutputInterface $output)
                 'See http://symfony.com/doc/current/cookbook/bundles/best_practices.html#bundle-name for more',
                 'details on bundle naming conventions.',
                 '',
-                'Use <comment>/</comment> instead of <comment>\\ </comment> for the namespace delimiter to avoid any problem.',
+                'Use <comment>/</comment> instead of <comment>\\ </comment>for the namespace delimiter to avoid any problems.',
```
Comment (xabbuh) [inline Command/GenerateBundleCommand.php]: What about removing the space inside the `comment` tag instead?

Comment (HeahDude) [inline Command/GenerateBundleCommand.php]: Does not work see the outdated diff with my comment

Comment (xabbuh) [inline Command/GenerateBundleCommand.php]: Shouldn't four backslashes do the trick then?

Comment (HeahDude) [inline Command/GenerateBundleCommand.php]: If you ask me I'd keep like this, but I've got no hard feeling :)

**Comment by fabpot** [comment]
Thank you @darrylhein.

File: Command/GenerateBundleCommand.php (hunk 2 above)
Comment (guilliamxavier) [inline Command/GenerateBundleCommand.php]: For reference: #141, #137.

**Pull request closed without merging**